lib/helium_config/core/lns.ex

```diff
@@ -0,0 +1,108 @@
+defmodule HeliumConfig.Core.Lns do
+  @moduledoc """
+  Utility functions for converting Core LNS models to and from JSON and DB representations.
+  """
+  alias HeliumConfig.Core
+  alias HeliumConfig.DB
+
+  def from_proto({:http_roaming, proto}) do
```

Comment:
I know you use this pattern of a module that implements `from_proto`, `from_web` and `from_db` functions extensively. I was just wondering if it would be worth codifying it into a behaviour? if that's not appropriate here than feel free to ignore, but thought it might lend a bit of extra structure around this pattern